core/domain/classifier_services.py

```diff
@@ -124,6 +124,35 @@ def classify_string_classifier_rule(state, normalized_answer):
 
     return None
 
+def check_classification_condition(state):
```

Comment:
In the subsequent PR, we are gonna be moving the classifier training away from the learner view and into a separate function train_classifier(). Inside this function, training will be done only if the necessary conditions (like Minimum training samples, and minimum labels) are satisfied. So, this function checks if the conditions for training are satisfied. @seanlip Could we name the function in a better way then?